portalnet/src/utils/db.rs

```diff
@@ -98,8 +99,13 @@ fn get_default_data_dir() -> anyhow::Result<PathBuf> {
 /// If the private key does not exist (eg. brand new trin data dir),
 /// a random pk is generated and stored.
 fn get_application_private_key(trin_data_dir: &Path) -> anyhow::Result<CombinedKey> {
-    let unsafe_private_key_file = trin_data_dir.join(UNSAFE_PRIVATE_KEY_FILE_NAME);
-    if !unsafe_private_key_file.exists() {
+    let trin_private_key_dir = match env::var(TRIN_PRIVATE_KEY_ENV_VAR) {
+        Ok(val) => PathBuf::from(val),
+        Err(_) => trin_data_dir.to_path_buf(),
```

Comment:
Ok it should be corrected now

env::var can return 2 things a error as in invalid environment input or it will tell you the enviroment variable isn't present, in the ladder case would would want to return the default trin path